tests/assets/deployment.template_2.json

```diff
@@ -29,7 +29,7 @@
           }
         },
         "modules": {
-          "temp-sensor-module": {
+          "tempSensor": {
```

Comment:
I used all lower because docker is all lower and I use that as the docker name.  Thoughts?

Reply:
@jongio The documents all reference to the temp sensor module as "tempSensor", for example https://docs.microsoft.com/en-us/azure/iot-edge/quickstart-linux#deploy-a-module and https://docs.microsoft.com/en-us/azure/iot-edge/tutorial-csharp-module#build-your-iot-edge-solution. For container names, Docker supports upper case. The edgeAgent and edgeHub also follows this style.

Reply:
@LazarusX  - This is what I get when I run build with mixed case

```
(venv) C:\temp>docker build . -t testFile
invalid argument "testFile" for "-t, --tag" flag: invalid reference format: repository name must be lowercase
See 'docker build --help'.
```

Reply:
Okay.